recipes-core/base-files/base-files/fstab.xenclient-dom0

```diff
@@ -4,6 +4,7 @@ devpts                        /dev/pts      devpts mode=0620,gid=5
 tmpfs                         /var/volatile tmpfs  defaults,rootcontext=system_u:object_r:var_t:s0,size=2M      0  0
 tmpfs                         /var/cache    tmpfs  defaults,rootcontext=system_u:object_r:var_t:s0,size=100M    0  0
 tmpfs                         /var/lib/dbus tmpfs  defaults,rootcontext=system_u:object_r:system_dbusd_var_lib_t:s0,size=1M    0  0
+tmpfs                         /var/lib/xen  tmpfs  defaults,rootcontext=system_u:object_r:xend_var_lib_t:s0,size=1M    0  0
```

Comment:
I assume this is another temp area xl needs. It seems we still have xend references all over the policy.